spec/lib/cds_importer/entity_mapper/base_mapper_spec.rb

```diff
@@ -0,0 +1,160 @@
+class MockedModel
```

Comment:
Since this is in the global namespace and will be available in other specs this model should probably be somewhere 'common' for specs - `spec/support` maybe?